lib/Doctrine/Validator.php

```diff
@@ -38,11 +38,14 @@ class Doctrine_Validator extends Doctrine_Locator_Injectable
      */
     private static $validators = array();
 
+    // This was undefined, added for static analysis and set to public so api isn't changed
+    public $stack;
```

Comment:
I couldn't find any other possible references to this from outside the class (the class itself sets it but never accesses it).